### ISSUE TITLE
Return SVG path data

### ISSUE THREAD

**Issue by emeeks** [pshihn/rough#51]
Would it be possible to add a feature that just returns SVG path data (just the `d` attribute of the fill and the `d` attribute of the stroke if any) instead of having `rough` append SVG nodes to the DOM. I'd find this easier to integrate into React that way.

**Comment by pshihn** [comment]
Thanks for the idea. This was already possible using [Generators](https://github.com/pshihn/rough/wiki/RoughGenerator) to some extent. But, I added a method specific to  to SVG Paths in the generator, that returns the SVG Path information needed. See: https://github.com/pshihn/rough/wiki/RoughGenerator#svg-path-info

A few notes:
* The paths must be rendered in the order they are returned. 
* Most paths are basic strokes, fills, etc. But for cases when using a hachure fill for an arbitrary path, instead of the fill property, a pattern property is returned. That's the pattern to use to fill the path. 

Hope this will make things easier for you. 

v2.1.1 should have it.  https://github.com/pshihn/rough/pull/52

**Comment by emeeks** [comment]
Yes I've already used it to fully integrate rough into my data viz library (replacing my old, simple sketchy function). It all makes perfect sense. You've done a really impressive job. Thanks!

<img width="534" alt="screen shot 2018-04-14 at 12 06 49 pm" src="https://user-images.githubusercontent.com/495634/38771695-68180966-3fdc-11e8-9d62-fd7ad7e14492.png">
<img width="634" alt="screen shot 2018-04-14 at 12 06 55 pm" src="https://user-images.githubusercontent.com/495634/38771696-6a07ead4-3fdc-11e8-803d-f12dd0a3451b.png">

**Comment by pshihn** [comment]
Thanks. Semiotic looks really cool btw. I've been meaning to give it a try. Just haven't been doing much dataviz work lately. :/
